internal/common/version.go

```diff
@@ -57,3 +58,15 @@ func BaseVersionEqual(version1, versionMayEqual string) (bool, error) {
 
 	return v1[0] == v2[0] && v1[1] == v2[1], nil
 }
+
+func GetBaseVersion(version string) (*string, error) {
```

Comment:
maybe we should add different test for different version formats?

Reply:
Will do thanks